data_io.py

```diff
@@ -0,0 +1,70 @@
+import mimetypes
+import os
+from json import dump
+
+import pandas as pd
+
+
+# Input
+def take_input_data(path: str) -> pd.DataFrame:
```

Comment:
wouldn't be better to name it load_input_data ?

Reply:
Good point, renamed.